tests/test_odgovlt.py

```diff
@@ -0,0 +1,48 @@
+from ckanext.harvest.harvesters.ckanharvester import CKANHarvester
+from ckanext.harvest.harvesters.ckanharvester import DatetimeEncoder
+from sqlalchemy import MetaData
+from sqlalchemy.engine.base import Engine
+from datetime import datetime
+
+
+def test_info():
+    ckan_object = CKANHarvester()
+    info = ckan_object.info()
+    assert isinstance(info, dict)
+    assert isinstance(info['name'], str)
+    assert isinstance(info['title'], str)
+    assert isinstance(info['description'], str)
+    assert isinstance(info['form_config_interface'], str)
+
+
+def test_connect_to_database(mocker):
+    ckan_object = CKANHarvester()
+    config = mocker.patch('ckanext.harvest.harvesters.ckanharvester.config')
+    config.__getitem__.side_effect = 'mysql://test'.split()
+    mocker.patch('ckanext.harvest.harvesters.ckanharvester.MetaData')
+    con, meta = ckan_object._connect_to_database()
+    assert isinstance(con, Engine)
+    mocker.stopall()
+    config = mocker.patch('ckanext.harvest.harvesters.ckanharvester.config')
+    config.__getitem__.side_effect = 'mysql://test'.split()
+    mocker.patch('ckanext.harvest.harvesters.ckanharvester.create_engine')
```

Comment:
I'm not sure why you stop all the mocks and do another round patching? Usually all the patching happens at the very top of the test function. And you only stop the mocks only if there is a good reason for that.